src/persist/src/file.rs

```diff
@@ -497,13 +497,28 @@ impl Blob for FileBlob {
         let file_path = self.core.blob_path(key)?;
         // TODO: strict correctness requires that we fsync the parent directory
         // as well after file removal.
+
+        fail_point!("fileblob_delete_before", |_| {
+            Err(Error::from(format!(
+                "FileBlob::delete before fail point reached for file {:?}",
```

Comment:
nit: can we change the error string to "FileBlob::delete_before failpoint reached for file ..."